src/app/ecosystem-explorer/page.tsx

```diff
@@ -73,40 +41,16 @@ export const metadata = createMetadata({
 })
 
 const categories = getEcosystemCMSCategories()
+const ecosystemProjects = getEcosystemProjectsData()
```

Comment:
Why not calling this function in `EcosystemExplorerContent`? Since we only call it here to pass is as a prop.

If so, we could do the same for events and projects.